### ISSUE TITLE
scammy sites found a new way to evade blacklists

### ISSUE THREAD

**Issue by graph** [StevenBlack/hosts#1147]
I noticed that it's easier recently to visit a scammy site by misspelling a legit domain. I have reported one a week ago and it still hasn't been added to any lists. One of such is whatsap.com I have already reported. One interesting aspect if you switch your user agent to google bot, it doesn't do anything sketchy. Upon looking like a normal user, it cycles from phishing, trying to get user to install malware, and other deceptive websites. I wouldn't be surprised if these new scammy websites serve different content based on IP address blocks aswell. It's not too hard to get googles IP address block and try to look not suspicious that way there by bypassing banning.

I don't know if this is new or not, but a week is too long of a wait since I have reported through firefoxes report button, and also googles website reporting. Thanks guys for making the web safer. I don't know where else to report such sites. In my previous experience websites were blocked in under a day from the time I have reported suspicious websites.

**Comment by StevenBlack** [comment]
Please provide a link to the issue you supposedly raised, because I can't find it.  

If you have an issue related to that issue, please post it in that issue, as a continuation, a friendly reminder.  Please don't be opening new issues complaining about some other issue without providing a link.  We're all busy people, here.

**Comment by graph** [comment]
You guys are great! I reported this issue here and it was added to matrix blacklist within a day. When I said I reported this issue before I was referring to reporting to firefox, & google security forms. I reported it to them and it wasn't banned at all, and still not banned. Sorry it sounds confusing, but you guys did a great job of adding it promptly to a black last.

Clearly the blacklists here are not connected to chrome, or firefox webbrowser as both browsers do not report that website as malicious. Usually in my experience it takes under a day for such websites to be blocked in my browser after reporting. But somehow it's evading them. I also sent an email recently to mozzilas security email to bring this more attention. I have a feeling that malicious websites have found a way to avoid the blacklists that major webbrowsers use, and my friend is also noticing that more malicious websites that are just typos are appearing.


You guys did a great job 👍
